policy/Object.yaml

```diff
@@ -0,0 +1,392 @@
+funcs:
+    objectId: # the index of deepCoadd_obj IS the objectId
+        functor: Index
+    parentObjectId:
+        functor: Column
+        args: parent
+        dataset: ref
+    coord_ra:
+        # reference position for merged "ref" cat. Required by db. Not in DPDD
+        functor: RAColumn
+        dataset: ref
+    coord_dec:
+        # reference position for merged "ref" cat. Required by db. Not in DPDD
+        # coord_dec because "dec" is reserved in most SQL DBs
+        functor: DecColumn
+        dataset: ref
+    Ra:
+        functor: RAColumn
+        dataset: meas
+    Dec:
+        functor: DecColumn
+        dataset: meas
+    # raErr:  not available yet DM-15180
+    # decErr: not available yet DM-15180
+    refBand: # reference band is same for all measurements
+        functor: ReferenceBand
+        dataset: ref
+    # psRadecTai: No moving point source model yet.
+    # psRadec: No moving point source model yet.
+    # psPm: No moving point source model yet.
+    # psParallax: No moving point source model yet.
+    PsFlux:
+        functor: NanoJansky
+        dataset: forced_src  # recommend meas too
+        args: slot_PsfFlux_instFlux
+    PsFluxErr:
+        functor: NanoJanskyErr
+        dataset: forced_src
+        args:
+            - slot_PsfFlux_instFlux
+            - slot_PsfFlux_instFluxErr
+    # Assume the database will provide UDFs for Mags.
+    # psCov:
+    #     Flux should be somewhere in the name?
+    #     How best to store cov matrix?
+    # psLnL:  No moving point source model yet.
+    # psChi2:  No moving point source model yet.
+    # psNdata: No moving point source model yet.
+
+    # DRP is working the the Bulge-Disk model fluxes
+    # In the meantime, CModel is the best
+    # CModel fluxes have alias slot_ModelFlux
+    #bdRadec: # Same as SdssCentroid
```

Comment:
Ditto re “d”

Space needed after #

Here and throughout, inconsistent use of period on comments.